config/spotbugs/exclude.xml

```diff
@@ -4,4 +4,34 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
 
+    <Match>
+        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient" />
+        <Method name="acknowledgeMessage" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE " />
+    </Match>
+
+    <Match>
+        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient" />
+        <Method name="authenticate" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE " />
+    </Match>
+
+    <Match>
+        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient" />
+        <Method name="getEdifactMessage" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE " />
+    </Match>
+
+    <Match>
+        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient" />
+        <Method name="getInboxMessageIds" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE " />
+    </Match>
+
+    <Match>
+        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient" />
+        <Method name="sendEdifactMessage" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE " />
+    </Match>
```

Comment:
Can these be combined? As the violation is the same for a group of methods within the same class. Maybe something like this:
```suggestion
    <Match>
        <Class name="uk.nhs.digital.nhsconnect.lab.results.mesh.http.MeshClient"/>
        <Or>
            <Method name="acknowledgeMessage"/>
            <Method name="authenticate"/>
            <Method name="getEdifactMessage"/>
            <Method name="getInboxMessageIds"/>
            <Method name="sendEdifactMessage"/>
        </Or>
        <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE "/>
    </Match>
```

Reply:
Done in 60a0a93